lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -264,15 +270,43 @@ def simpler_gff3(self, gff3_in: TextIO, out_gff_path: Path, out_funcann_path: Pa
                 new_records.append(new_record)
             
             if fail_types and not skip_unrecognized:
-                raise Exception("Unrecognized types found (%s): fail" %
-                                (" ".join(fail_types.keys())))
+                raise Exception(f"Unrecognized types found ({' '.join(fail_types.keys())}): fail")
             
             GFF.write(new_records, gff3_out)
         
         # Write functional annotation
         functional_annotation = self.clean_functional_annotations(functional_annotation)
         print_json(out_funcann_path, functional_annotation)
 
+    def format_mobile_element(self, feat, functional_annotation):
+        """Given a mobile_genetic_element feature, transform it into a transposable_element"""
+        quals = feat.qualifiers
+
+        if feat.type == 'mobile_genetic_element':
+            mobile_element_type = feat.qualifiers.get("mobile_element_type", [])
+            if mobile_element_type:
+                element_type, transposon_type = mobile_element_type[0].split(':')
+                if element_type in ("transposon", "retrotransposon"):
+                    feat.type = 'transposable_element'
+                    if not feat.qualifiers.get("product"):
```

Comment:
could you post some examples here? Thanks

Reply:
From GCA_002219185.1 (C. glabrata DSY562):
`CM008085.1      Genbank mobile_genetic_element  1575    7277    .       +       .       ID=id-CM008085.1:1575..7277;Note=label: mobile_element;gbkey=mobile_element;mobile_element_type=retrotransposon;rpt_type=long_terminal_repeat`

From GCA_026262505.1 (T. vaginalis):
`CM044617.1      Genbank mobile_genetic_element  299     2423    .       -       .       ID=id-CM044617.1:299..2423;gbkey=mobile_element;mobile_element_type=transposon:rnd-1_family-319/TvMULE1`

(Note how the first has only the element type, and the second also has a name)